app/controllers/build_commands_controller.rb

```diff
@@ -10,14 +10,10 @@ def show
 
   def update
     command = params[:command][:command]
-    if command.blank?
```

Comment:
@grosser If we just do `destroy!` and let it blow up then the only way the user can have the project not do anything before building the docker image is to save a null command. For example, `/* do nothing */`. See point 3 of https://github.com/zendesk/samson/pull/2186#issuecomment-322545884.

I'm fine with any of those options. I just want to make it clear how we intend users to use Samson. What do you think?